pkg/cli/containers.go

```diff
@@ -39,11 +40,23 @@ func (a *Container) Run(cmd *cobra.Command, args []string) error {
 	out := table.NewWriter(tables.Container, a.Quiet, a.Output)
 
 	if len(args) == 1 {
-		app, err := c.ContainerReplicaGet(cmd.Context(), args[0])
+		app, err := c.AppGet(cmd.Context(), args[0])
 		if err != nil {
-			return err
+			// see if it's the name of a container instead
+			container, err := c.ContainerReplicaGet(cmd.Context(), args[0])
+			if err != nil {
+				return err
+			}
+			out.Write(container)
+		} else {
+			containers, err := c.ContainerReplicaList(cmd.Context(), &client.ContainerReplicaListOptions{App: app.Name})
+			if err != nil {
+				return err
+			}
+			for _, container := range containers {
+				out.Write(container)
```

Comment:
Sorry, one more comment. Should we make this output similar to the output below in that stopped containers are only printed if the `-a` flag is passed?

Reply:
I'm not 100% sure. It looks like the case for that below is when the command run is simply `acorn containers` at which point it is pretty much the same as just `acorn ps`. I can make it behave that way if that is what would be best.

Reply:
Yeah, let's make `-a` consistently the same
- `acorn containers` - all running containers
- `acorn containers -a` - all running and stopped containers
- `acorn containers my-app` - all running containers belonging to my-app
- `acorn containers -a my-app` - all running and stopped containers belong to my-app

So (probably obvious), you'll need to add the `a.All || container.Status.Columns.State != "stopped" ` check